### PR TITLE
Fixed #33: <br/> in ncurses RichText did not behave correctly.

### LINKED ISSUE #33: Different behavior on RichText widget
html tags like <br/> and <font color=...> are accepted but the behavior compared to Qt and Gtk is different. In the first case the "br" is ignored and next text is on the same line, for the latter the color 
is always the same. Both cases work well -or at least as expected- on Gtk and Qt.

### PR DESCRIPTION
**Pull request by gilsonsouza**

### CODE REVIEW

**Comment by jreidinger** [comment]
in general it looks good, please add also changelog entry and bump version.

File: src/NCRichText.cc [hunk 1]
```diff
@@ -887,6 +888,7 @@ bool NCRichText::PadTOKEN( const wchar_t * sch, const wchar_t *& ech )
 
 	    if      ( value == L"big" )		token = T_IGNORE;
 	    else if ( value == L"pre" )		token = T_PLAIN;
+	    else if ( value == L"br/" )		token = T_BR;
```
Comment (gilsonsouza) [inline src/NCRichText.cc]: Just to be sure: Should I add such info as a comment in the code?

Comment (gilsonsouza) [inline src/NCRichText.cc]: I look all tags and the only other non-pair tag that I found was <hr>. 
I added the if statement to set token to T_IGNORE in <hr/> case, as it already does in <hr> case.

**Comment by jreidinger** [comment]
for changelog I recommend to use `osc vc` command

**Pull request merged**